v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/SpannerToBigQuery.java

```diff
@@ -54,7 +54,13 @@ public static void main(String[] args) {
   }
 
   private static BigQueryIO.Write<String> writeToBigQuery(SpannerToBigQueryOptions options) {
-    return BigQueryConverters.<String>createWriteTransform(options)
+    return BigQueryIO.<String>write()
+        .to(options.getOutputTableSpec())
+        .withWriteDisposition(
+            BigQueryIO.Write.WriteDisposition.valueOf(options.getWriteDisposition()))
```

Comment:
Please import `WriteDisposition` and `CreateDisposition`. In general, nested classes/enums should be imported if able, since it makes code more readable.